File: Java/Program/Monitor.java
public class Monitor extends Periferal 
{
    private int ukuranInci;
    private int refreshRate;

    public Monitor(String merk, String nama, String koneksi, int ukuranInci, int refreshRate) 
    {
        super(merk, nama, koneksi);
        this.ukuranInci = ukuranInci;
        this.refreshRate = refreshRate;
    }

    public void setUkuranInci(int ukuranInci) 
    {
        this.ukuranInci = ukuranInci;
    }

    public int getUkuranInci() 
    {
        return ukuranInci;
    }

    public void setRefreshRate(int refreshRate) 
    {
        this.refreshRate = refreshRate;
    }

    public int getRefreshRate() 
    {
        return refreshRate;
    }
}

File: Python/Program/Monitor.py
from Periferal import Periferal

class Monitor(Periferal):
    def __init__(self, merk, nama, koneksi, ukuranInci, refreshRate):
        super().__init__(merk, nama, koneksi)
        self.ukuranInci = ukuranInci
        self.refreshRate = refreshRate
        
    def setUkuranInci(self, ukuranInci):
        self.ukuranInci = ukuranInci
        
    def getUkuranInci(self):
        return self.ukuranInci
    
    def setRefreshRate(self, refreshRate):
        self.refreshRate = refreshRate
        
    def getRefreshRate(self):
        return self.refreshRate
    